typescript/package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "digital-factory-sdk",
+  "version": "1.0.0",
+  "description": "Try out the Ultimaker Digital Factory API",
+  "main": "main.ts",
+  "scripts": {
+    "start": "tsc && node dist/main.js"
+  },
+  "author": "Ultimaker <software-cloud@ultimaker.com>",
+  "license": "ICS",
+  "dependencies": {
```

Comment:
Are we going to set up dependabot for this repo?

Reply:
I don't think that is needed here.